src/chopper_count.cpp

```diff
@@ -12,56 +12,86 @@ namespace chopper::count
 
 void initialize_argument_parser(seqan3::argument_parser & parser, chopper::count::configuration & config)
 {
-    parser.info.author = "Avenja";
-    parser.info.short_description = "Count all kmers of each file in a directory.";
     parser.info.version = "1.0.0";
+    parser.info.author = "Svenja Mehringer";
+    parser.info.email = "svenja.mehringer@fu-berlin.de";
+    parser.info.short_description = "Estimate the (representative) k-mer count";
 
+    parser.info.description.emplace_back("Estimates the (representative) k-mer count of sequence data by computing "
+                                         "HyperLogLog sketches [1] of the input data. "
+                                         "Additionally, the HLL sketches are stored in a directory (if not disabled) "
+                                         "and can be used in computing an HIBF layout with copper layout.");
```

Comment:
```suggestion
                                         "Additionally, the HLL sketches are stored in a directory "
                                         "and can be used in computing an HIBF layout with chopper layout.");
```

On re-reading, I think we do not need to mention here that one can disable the sketch output.